nodejs/week1/homework/warmup.js

```diff
@@ -0,0 +1,19 @@
+console.log("inside warmup file");
+
+class Circle {
+    constructor(radius) {
+        this.radius = radius;
+    }
+    getDiameter() {
+        console.log(this.radius * 2);
```

Comment:
```suggestion
const diameter = this.radius * 2
 console.log(diameter)
return diameter
```
The functions should return a value for this exercise.

Reply:
Thanks Florin.. Will implement the changes you mentioned.